### ISSUE TITLE
PIA and port forwarding

### ISSUE THREAD

**Issue by Khabarakh** [haugene/docker-transmission-openvpn#1411]
Hi all, i've got a problem, i don't know if it's me or the soft.
i'm using PIA VPN and i can't find a way to make it work with port forwarding, each time transmission say that the port are closed, so it don't work well. maybe someone can help me. ( forgive my bad english.)

**Comment by mjvotaw** [comment]
I've also been having issues recently with getting a port from PIA. I think they (PIA) are getting ready to start pushing people to use their Next Generation servers. They've got some info on their blog [here](https://www.privateinternetaccess.com/helpdesk/news/posts/decommissioning-of-legacy-network) and [here](https://www.privateinternetaccess.com/helpdesk/news/posts/important-updated-server-changes-and-related-issues).
Last week they released some scripts for manually connecting to the Next Gen servers and setting up port forwarding: [Manual Connection and Port Forwarding Scripts](https://www.privateinternetaccess.com/helpdesk/kb/articles/manual-connection-and-port-forwarding-scripts).  I've actually spent a little while trying out PIA's new scripts, and I don't think this container is setup in a way that will let us easily connect to their new vpn servers.

**Comment by haugene** [comment]
Check out #1334
You're right @mjvotaw that this has to do with the next gen servers. A new port forwarding script is merged to the dev branch and can be tested using the `dev` tag of the image. Last I saw PIA was planning to shut down the current servers by the end of the month so this should be merged to master within that time. As of now we're testing it out so feel free to join the testing and report any issues in the issue I linked to.

If I'm not missing anything I think this thread can be closed as a duplicate.
